projects/driver_controls/test/test_ads1015.c

```diff
@@ -58,28 +59,28 @@ void test_ads1015_init_invalid_input(void) {
                     ads1015_init(&s_storage, TEST_ADS1015_I2C_PORT, TEST_ADS1015_ADDR, NULL));
 }
 
-void test_ads_config_channel_invalid_input(void) {
+void test_ads1015_config_channel_invalid_input(void) {
   // Tests a basic use of the function
   TEST_ASSERT_EQUAL(STATUS_CODE_OK, ads1015_configure_channel(&s_storage, ADS1015_CHANNEL_0, true,
-                                                              prv_callback, &s_storage));
+                                                              prv_callback_channel, &s_storage));
```

Comment:
You shouldn't use `s_storage` - if your check was removed and the callback actually ran, `s_storage` would be modified.

Reply:
Ah, I dont even know why the context is s_storage, it is supposed to be those booleans for channels.
Thank you.